files/etc/uci-defaults/99-nextsec-netifyd

```diff
@@ -0,0 +1,7 @@
+if [ -f /usr/lib/libnetify-flow-actions.so.0.0.0 ] && ! grep -q np-nfa /etc/netifyd.conf; then
```

Comment:
```suggestion
if [ -f /usr/lib/libnetify-flow-actions.so.0 ] && ! grep -q np-nfa /etc/netifyd.conf; then
```

Reply:
I do not get this one.
`/usr/lib/libnetify-flow-actions.so.0` is a link to `/usr/lib/libnetify-flow-actions.so.0.0.0`, why testing the presence of the link and then adding the link target to the configuration file?
I think it's not coherent.

Reply:
I do not get this one.
`/usr/lib/libnetify-flow-actions.so.0` is a link to `/usr/lib/libnetify-flow-actions.so.0.0.0`, why testing the presence of the link and then adding the link target to the configuration file?
I think it's not coherent.

Reply:
Using the link should avoid configuration changes on minor releases.